scripts/render.js

```diff
@@ -0,0 +1,25 @@
+import fs from "fs-extra";
+import glob from "glob";
+import path from "path";
+import { getNameAndSize } from "./util/helpers.js";
+
+const svgPaths = glob.sync(`./dist/**/*.svg`);
+
+export const iconsBySize = () => {
+  const iconsBySize = {};
+  const icons = svgPaths.map((svgPath) => {
+    const data = fs.readFileSync(svgPath, "utf-8");
+    const {size} =  getNameAndSize(svgPath);
+    return {
+      name: path.parse(svgPath).name,
```

Comment:
Can we show the "icon name" instead of just "icon_16": 
<img width="306" alt="image" src="https://github.com/warp-ds/icons/assets/5851332/20be2ec7-6cba-470c-a5da-9fcd9941fe28">
<img width="310" alt="image" src="https://github.com/warp-ds/icons/assets/5851332/6d7c7461-e933-4b4e-baa2-8709c2dadc74">
Something like this for example:

```suggestion
const regex = /dist\/(.*?)\/icon/;
    const match = svgPath.match(regex);
    let iconName = path.parse(svgPath).name;
    if (match && match[1]) {
      iconName = match[1];
    }

    return {
      name: iconName,
```